xds/internal/balancer/edsbalancer/balancergroup.go

```diff
@@ -48,8 +60,24 @@ type subBalancerWithConfig struct {
 	balancer balancer.Balancer
 }
 
-func (sbc *subBalancerWithConfig) startBalancer(bgcc *balancerGroupCC) {
-	b := sbc.builder.Build(bgcc, balancer.BuildOptions{})
+// UpdateBalancerState is overridden to keep state and picker.
+func (sbc *subBalancerWithConfig) UpdateBalancerState(state connectivity.State, picker balancer.Picker) {
```

Comment:
Now, `balancerGroupCC` doesn't have much functionality at all. Does it make sense to move that in here so that `subBalancerWithConfig` (we would probably need a better name, though) implements the full `balancer.ClientConn` interface for the sub-balancers?

Reply:
Good catch. Done.

But didn't rename. Can't pick a good name.